README.md

```diff
@@ -1 +1,15 @@
 # hw01
+
+## Self Introduction
+My name is Yuwen Xiong. I am a second-year master student at **UChicago Harris School of Public Policy**. Below is the photo of mine:
+
+![Photo of Yuwen](LinkedIn-profile-smaller.jpg)
+
+If you want to know more about me, you can take a look at [my LinkedIn profile](https://www.linkedin.com/in/yuwenxiong/) .
+
+### Interesting Facts about me
+* I enjoy playing tennis.
+* My favorite musical is French musical *Romeo et Juliet*.
+* I am worried about finding a job.
+
+It is a great pleasure to meet everyone.
```

Comment:
Great to meet you too! I think you did a great job on this assignment - one thing I noticed was that there were not a lot of commits in the file to git; in future assignments maybe committing more often can make sure you don't lose any work and are able to better keep track of changes!

Reply:
I will pay attention to that! Thank you so much!